apicapi/apic_client.py

```diff
@@ -297,8 +305,12 @@ def __init__(self, hosts, usr, pwd, ssl, verify=False,
         self.authentication = None
         self.username = None
         self.password = None
+        self.cert_name = None
+        self.private_key = None
         if usr and pwd:
             self.login(usr, pwd)
+        elif usr and cert_name and private_key_file:
```

Comment:
in the configuration file, do we need to explicitly set the user and password to None to make this work?
I know that configuration is out of apicapi, but I would like to take a note for a future fix if that's the case.